content/teaching-curriculum/introducing-teach-like-a-champion.md

```diff
@@ -0,0 +1,29 @@
+---
+_db_id: 
+content_type: topic
+flavours:
+- teaching
+learning_outcomes: null
+prerequisites:
+  hard: []
+  soft: []
+ready: true
+tags: 
+title: Introducing Teach Like A Champion
+---
+
+Welcome to our 
+
+## Intro to Teach like a Champion
+
+
+## Quick Familiarty Projects
+Quick high level overview > awareness > just to start noticing things you otherwise wouldn't. 
+
+## Chapter Deep Dives
+Each Chapter > a bunch of techniques with the same goal. 
+It takes more than awareness > it takes practice/application. 
+>>> Do some planning
+
+## Repeated application projects
+Deliberate practice requires coaching. >>> Let's apply this lens in multiple places, and get feedback on our thinking/understanding of it. 
```

Comment:
I'm not really sure what this is for. Topics are meant to be self-study things. Most topics are useful things that a person might refer back to. 

Is this a description of a project? Maybe this text should be in the project? 

It is possible that I'm completely misunderstanding the intention of this file though. I worry that other people will too.